includes/unsupported-theme-handlers/class-sensei-unsupported-theme-handler-page-imitator.php

```diff
@@ -85,6 +85,8 @@ protected function output_content_as_page( $content, $object_to_copy = null, $po
 		$wp_query        = clone $wp_query;
 		$wp_query->post  = $post;
 		$wp_query->posts = array( $post );
+		$wp_query->queried_object = $post;
```

Comment:
We should align our assignments in this block. Not sure why the linter didn't pick up on this.

Reply:
Oh, the linter _did_ pick up on it, I just hadn't noticed 😆 

So yeah, this is what CI is complaining about 🙂